### DISCUSSION TITLE
RT5 interop for symmetric irb

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#490] [General]
SR Linux follows [RFC9136 for ip prefix rt5 announcements](https://www.rfc-editor.org/rfc/rfc9136.html#name-interface-less-ip-vrf-to-ip), and does not support RFC9135 (which involves 2 RTs, one for l2 and a second for l3)
In [FRR terms](https://docs.nvidia.com/networking-ethernet-software/cumulus-linux-41/Network-Virtualization/Ethernet-Virtual-Private-Network-EVPN/Inter-subnet-Routing/#evpn-type-5-routing-in-asymmetric-mode) this corresponds to the ```prefix-routes-only``` option

Consequently, inter-subnet forwarding does not work as one would expect. How should we handle/flag this, if anything?

**Comment by ipspace** [comment]
Citing the NVIDIA document you linked to:

> Cumulus Linux supports EVPN type-5 routes for prefix-based routing in asymmetric configurations within the pod or data center by providing an option to use the layer 3 VNI only for type-5 routes

It still uses layer-3 VNI (there's no magic), it's just that it performs intra-subnet bridging instead of intra-subnet routing. It's really the question of "do we bridge or route within the egress directly-connected subnet". Looks like another blog post is in order ;)

**Comment by ipspace** [comment]
The way I read that RFC, the devices that do not support intra-subnet IP forwarding can ignore those announcements, so I guess we don't have to worry about that.
